meta/acronyms.txt

```diff
@@ -55,7 +55,7 @@ RSPAN - Remote Span
 SAI - Switch Abstraction Interface
 SDK - Software Development Kit
 SG
-SNMP - Services and Protocols for Advanced Networks
```

Comment:
in saiswitch.h "Both *G/SG lookup supported" not familiar with technology no clue what SG mean :)